03-setup-ingress-controller/ambassador.md

```diff
@@ -227,7 +329,7 @@ Explanations for the above configuration:
 - In general the `registrant email address` is mandatory when using `ACME`, and it should be a valid one in order to receive notifications when the certificates are going to expire.
 - The Ambassador Edge Stack built-in `ACME` client knows to handle `HTTP-01` challenges only. For other `ACME` challenge types like `DNS-01` for example, an `external` certificate management tool is required (e.g. [Cert-Manager](https://cert-manager.io)).
 
-For more details, please visit the [AES Host CRD](https://www.getambassador.io/docs/edge-stack/1.13/topics/running/host-crd/) official documentation.
+For more details, please visit the [AES Host CRD](https://www.getambassador.io/docs/edge-stack/2.1/topics/running/host-crd/) official documentation.
```

Comment:
Please leave only the `Host` string for the link (I know that this was there before your changes, so it's ok).

Reply:
Thank you, updated the line